src/main/java/uk/gov/justice/probation/courtcaseservice/controller/OffenderMatchesController.java

```diff
@@ -48,28 +53,38 @@ Mono<ResponseEntity<Object>> createGroupedOffenderMatchesByDefendant(@PathVariab
     @GetMapping(value = "/case/{caseId}/defendant/{defendantId}/grouped-offender-matches/{groupId}", consumes = APPLICATION_JSON_VALUE, produces = APPLICATION_JSON_VALUE)
     @ResponseStatus(HttpStatus.OK)
     public @ResponseBody
-    Mono<GroupedOffenderMatchesEntity> getOffenderMatchesByCaseId(@PathVariable(value = "caseId") String caseId,
-                                                                  @PathVariable(value = "defendantId") String defendantId,
-                                                                  @PathVariable(value = "groupId") Long groupId) {
+    Mono<GroupedOffenderMatchesEntity> getOffenderMatchesByCaseId(@PathVariable(value = CASE_ID_PARAM) String caseId,
```

Comment:
I'm not sure why we've specified the `@PathVariable` values here at all, they're redundant as the parameter name already matches what's in the URL so I'd get rid of them if we can.

Reply:
Agree, will remove them all.